core/src/components/select/select.ionic.scss

```diff
@@ -9,14 +9,18 @@
   --background: #{globals.$ion-primitives-base-white};
   // TODO(ROU-10778, ROU-10875): Sync the color names to the design system of
   // ios and md. This will allow us to have a single color map.
-  --border-color: #{globals.current-color(neutral)};
+  --border-color: #{globals.$ion-primitives-neutral-500};
```

Comment:
Why was this value changed? The ticket is supposed to be for focus only. I would recommend making a separate PR.

Reply:
I explain my reasons on the PR description. As this was such a simple fix for a obvious bug and with impacts on focus visibility on snapshots, I thought we could include it here. Its all related related to changes on select border color, after all. 

Can we keep the changes? If you still feel it makes no sense, I can do it separately yes 👍